$OpenBSD: patch-src_s_openbsd_h,v 1.1.1.1 2011/03/28 15:18:56 jasper Exp $

emacs goes into a loop where it is expecting to get a signal-driven
I/O but the signal is never delivered.
Defining BROKEN_SIGIO makes it use poll exclusively and fixes the
problem.

--- src/s/openbsd.h.orig	Tue Mar 15 16:33:00 2011
+++ src/s/openbsd.h	Tue Mar 15 16:34:00 2011
@@ -36,5 +36,7 @@
 
 #endif
 
+#define BROKEN_SIGIO
+
 /* arch-tag: 7e3f65ca-3f48-4237-933f-2b208b21e8e2
    (do not change this comment) */
